index.js

```diff
@@ -1,27 +1,50 @@
-let books = [];
+// class Books {
+//   constructor(books = []) {
+//     this.list = books;
+//   }
+// }
 
-const saveData = () => {
-  localStorage.setItem('myBooks', JSON.stringify(books));
-};
+// class Book {
+//   constructor(title, author, id) {
+//     this.title = title;
+//     this.author = author;
+//     this.id = id;
+//   }
+// }
 
-const addBook = (title, author) => {
-  const awesomeBook = {
-    title,
-    author,
-    id: books.length + 1,
+class Methods {
+  constructor(title, author, id) {
+    this.title = title;
+    this.author = author;
+    this.id = id;
```

Comment:
- [ ] You are not making use of these properties, please take a look at the previous comments, and organize your code better. 😄